data-manipulation/compression/create-cabinet-file.yml

```diff
@@ -0,0 +1,23 @@
+rule:
+  meta:
+    name: create Cabinet file
+    namespace: data-manipulation/compression
+    authors:
+      - michael.hunhoff@mandiant.com
+      - jakub.jozwiak@mandiant.com
+    scope: function
+    att&ck:
+      - Collection::Archive Collected Data::Archive via Library [T1560.002]
+    mbc:
+      - Data::Compress Data [C0024]
+    references:
+      - https://learn.microsoft.com/en-us/windows/win32/devnotes/creating-a-cabinet
+    examples:
+      - 44bad2e2a9e387b86870f009d01833ea4618d2a7cda5f64fa84a19f3bdf4efaf:0x1400028E0
+  features:
+    - and:
+      - match: create File Compression Interface context
+      - or:
+        - api: cabinet.FCIAddFile = add file to Cabinet
+        - api: cabinet.FCIFlushFolder = flush current folder under construction
+        - api: cabinet.FCIFlushCabinet = completes current cabinet
```

Comment:
Add `cabinet.FCIDestroy`?

Reply:
Thanks! Added in https://github.com/mandiant/capa-rules/pull/808/commits/ac09516cca968e54d6b5764405337cba3f290c21